### PR TITLE
add Doc

### PR DESCRIPTION
**Pull request by lchu-ibm**
Add README/Doc to the repo.

### CODE REVIEW

**Review by nairbv — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
...
+The generated model has a good performance on various metrics as evaluated by [lm-evaluation-harness](https://github.com/EleutherAI/lm-evaluation-harness), with MMLU score of 0.5. We share further [scores](docs/evaluation.md) in the details of the model for completeness.
+
+### Dataset
+We use an internally curated dataset, Blue Pile for training the model. We use sampling ratios similar to what Llama1 paper proposed with minor changes (e.g., no C4 dataset). Since the goal of this repo is to demonstrate the feasibility of training using PyTorch components at scale, we omit the details of the sampling ratios. The overall dataset is roughly 1.5T tokens and the model has seen all the tokens in the dataset at least once.
```
Comment (nairbv) [inline README.md:31]: maybe an additional `,` after "Blue Pile" or could reword to "For training the model we used ... called blue pile"

File: docs/dataloader.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Data Loader
+
+## Background
+
```
Comment (nairbv) [inline docs/dataloader.md:4]: might be better to break this into paragraphs or separate lines (partly just so easier to comment on details in a PR)

Comment (nairbv) [reply docs/dataloader.md:4]: we might want to add some additional info on things like that metadata is used to overcome file system limitations, that the dataset loads the data from cos/s3, etc

Comment (lchu-ibm) [reply docs/dataloader.md:4]: The dataset part was contributed by @daviswer .

@daviswer , can feel free to modify `dataloader.md` in place if you want to add anything here.

Comment (daviswer) [reply docs/dataloader.md:4]: Didn't realize that writeup was going here! I added pgraph breaks and a pointer to the `dataset_utils.py` file itself for further detail.

**Review by raghukiran1224 — APPROVED** [review]
lgtm

**Pull request merged**